mzroll/eicwidget.cpp

```diff
@@ -151,6 +151,14 @@ void EicWidget::integrateRegion(float rtmin, float rtmax) {
 	eicParameters->_integratedGroup.minQuality = settings->value("minQuality").toDouble();
 	eicParameters->_integratedGroup.compound = eicParameters->_slice.compound;
 	eicParameters->_integratedGroup.srmId = eicParameters->_slice.srmId;
+	eicParameters->_integratedGroup.samples.clear();
```

Comment:
@IpankajI This piece of code is already being in EIC.cpp. Is it possible to refactor this?